src/server/render/default-document/index.js

```diff
@@ -21,7 +22,9 @@ const escapeScriptTags = data => {
 const getProductionBundle = () => {
   const assetsPath = path.resolve(process.cwd(), '.tapestry', 'assets.json')
   const assets = fs.readJsonSync(assetsPath)
-  return assets.client.js
+  return map(assets, asset => (
```

Comment:
Hatin on lodash atm. Could we use `Object.values()` here? `return Object.values(assets).map(({ js })=> '<script src={js} />)`

Reply:
Now that's what I call ES2015